composer.json

```diff
@@ -21,11 +21,10 @@
     ],
     "require": {
         "php": "^7.2|^8.0",
-        "symfony/framework-bundle": "~3.4|~4.0|~5.0|~6.0",
```

Comment:
Should be reverted. Otherwise Composer would refuse to install the bundle with Symfony 6 again.

Reply:
It's just here to demonstrate how it's possible to test versions that are not supported officially. Will be reverted 👍